packages/flutter/test/material/slider_theme_test.dart

```diff
@@ -1218,6 +1218,123 @@ void main() {
 
     await gesture.up();
   });
+
+  testWidgets('activeTrackRadius is taken into account when painting the border of the active track', (WidgetTester tester) async {
+    await tester.pumpWidget(_buildApp(
+      ThemeData().sliderTheme.copyWith(
+        trackShape: const RoundedRectSliderTrackShapeWithCustomAdditionalActiveTrackHeight(
+          additionalActiveTrackHeight: 10.0
+        )
+      )
+    ));
+    await tester.pumpAndSettle();
+    final Offset center = tester.getCenter(find.byType(Slider));
+    await tester.startGesture(center);
+    expect(
+      find.byType(Slider),
+      paints
+        ..rrect(rrect: RRect.fromLTRBAndCorners(
+          24.0, 293.0, 24.0, 307.0,
+          topLeft: const Radius.circular(7.0),
+          bottomLeft: const Radius.circular(7.0),
+        ))
+        ..rrect(rrect: RRect.fromLTRBAndCorners(
+          24.0, 298.0, 776.0, 302.0,
+          topRight: const Radius.circular(2.0),
+          bottomRight: const Radius.circular(2.0),
+        )),
+    );
+  });
+
+}
+
+class RoundedRectSliderTrackShapeWithCustomAdditionalActiveTrackHeight extends RoundedRectSliderTrackShape {
+  const RoundedRectSliderTrackShapeWithCustomAdditionalActiveTrackHeight({this.additionalActiveTrackHeight});
+  final double? additionalActiveTrackHeight;
+  @override
+  void paint(
+    PaintingContext context,
+    Offset offset, {
+    required RenderBox parentBox,
+    required SliderThemeData sliderTheme,
+    required Animation<double> enableAnimation,
+    required TextDirection textDirection,
+    required Offset thumbCenter,
+    bool isDiscrete = false,
+    bool isEnabled = false,
+    double additionalActiveTrackHeight = 2.0,
```

Comment:
This is testing the test code.
You should do something like this
```dart
void paint(
    PaintingContext context,
    Offset offset, {
    required RenderBox parentBox,
    required SliderThemeData sliderTheme,
    required Animation<double> enableAnimation,
    required TextDirection textDirection,
    required Offset thumbCenter,
    bool isDiscrete = false,
    bool isEnabled = false,
}) {
    super.paint(......., additionalActiveTrackHeight: additionalActiveTrackHeight);
}
```

Reply:
Ah yes, now I really feel embarrassed for copying the whole code. I'll make the changes right away :)